hedera-node/hedera-consensus-service-impl/src/main/java/com/hedera/node/app/service/consensus/impl/handlers/ConsensusUpdateTopicHandler.java

```diff
@@ -177,6 +189,9 @@ private void resolveMutableBuilderAttributes(
         builder.expirationSecond(resolvedExpiryMeta.expiry());
         builder.autoRenewPeriod(resolvedExpiryMeta.autoRenewPeriod());
         builder.autoRenewAccountId(resolvedExpiryMeta.autoRenewAccountId());
```

Comment:
Can we keep line 191 in else case  on line 194 just for readability ?

Reply:
line 191 is still there, not removed. line 192-194 is to overwrite the value if the condition is true.

Reply:
Updated.